backend/Utils/Edna.Utils.Http/Edna.Utils.Http/EdnaExternalHttpHandler.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Net.Http;
+using System.Security.Cryptography.X509Certificates;
+using System.Net.Security;
+
+namespace Edna.Utils.Http
+{
+    public class EdnaExternalHttpHandler: HttpClientHandler
+    {
+        public static string Name = nameof(EdnaExternalHttpHandler);
```

Comment:
Shouldn't this be `const` instead?

Reply:
I agree. I was trying to make the implementation as consistent as possible with the one done for `EdnaHttpHandler`. But making it const makes more sense, yes. I will update this change.